page-components/featured-content-module.md

```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
 
 <div class="content-67 content-first">
 
-Featured content modules should include a single piece of content from a defined content type, including: Blog, News, Video, Event, and Tool.
+Featured content modules should highlight a single piece of content. That content can be almost anything, from a webpage to a blog post to a PDF to a video.
 
 FCMs appear below the title and introductory paragraph on a page, and retain that position on all screen sizes.
 
 FCMs contain a headline, body copy text, and a graphical element related to the piece of content being featured. They are a “mini story” and never simply a dominant visual.
 
-FCMs can appear on any page type, though they are best suited for pages 3rd level and below in the site map. They are designed to accommodate pages with either a left-hand navigation or right-hand sidebar.
+FCMs can appear on any page type, though they are best suited for pages third level and below in the site map. They are designed to accommodate pages with either a left-hand navigation or right-hand sidebar.
 
 </div>
 
 <div class="content-33 content-last"></div>
 
 ## Style at desktop size
 
+{::nomarkdown}
 <div>
-![Image of featured content module example at desktop size]({{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png)
+  <img src="{{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png" alt="Image of featured content module example at desktop size">
```

Comment:
Did you do this to get it out of the `<p>` tag that was constraining its width? Smart!

Reply:
Yep, that seemed to be the only way I could get the image to be big enough to see well.